packages/core-bridge/index.d.ts

```diff
@@ -98,9 +112,18 @@ export interface OtelCollectorExporter {
      * Optional set of HTTP request headers to send to Collector (e.g. for authentication)
      */
     headers?: Record<string, string>;
+    /**
+     * Specify how frequently in metrics should be exported.
+     *
+     * @format number of milliseconds or {@link https://www.npmjs.com/package/ms | ms-formatted string}
+     * @defaults 1 second
+     */
+    metricPeriodicity?: string | number;
```

Comment:
Maybe use plural?

```suggestion
    metricsPeriodicity?: string | number;
```

Reply:
TBH I'd name it `metricsExportInterval`, it reads better than Periodicity - a term I encountered here for the first time.